docs/core/tools/dotnet-test.md

```diff
@@ -64,6 +64,10 @@ Test projects specify the test runner using an ordinary `<PackageReference>` ele
 - **`--collect <DATA_COLLECTOR_FRIENDLY_NAME>`**
 
   Enables data collector for the test run. For more information, see [Monitor and analyze test run](https://aka.ms/vstest-collect).
+  
+  To collect code coverage on any platform that is supported by .NET Core, install [Coverlet](https://github.com/tonerdo/coverlet/blob/master/README.md) and use the `--collect:"XPlat Code Coverage"` option.
```

Comment:
let's use new link to coverlet:
https://github.com/coverlet-coverage/coverlet/blob/master/README.md

Reply:
```suggestion
  To collect code coverage on any platform that is supported by .NET Core, install [Coverlet](https://github.com/coverlet-coverage/coverlet/blob/master/README.md) and use the `--collect:"XPlat Code Coverage"` option.
```